docs/content/toc.md

```diff
@@ -42,6 +42,7 @@
     * [Context](../querying/query-context.html)
   * [Multi-value dimensions](../querying/multi-value-dimensions.html)
   * [SQL](../querying/sql.html)
+  * [Lookups](../querying/lookups.html)
   * [Joins](../querying/joins.html)
```

Comment:
@vogievetsky mentioned it was difficult to find in there since he was just ctrl-F for "lookups"